src/backend/aspen/test_infra/models/usergroup.py

```diff
@@ -47,7 +50,7 @@ def user_factory(
     system_admin=False,
     agreed_to_tos=True,
 ) -> User:
-    return User(
+    user = User(
```

Comment:
Will this function still be used after this PR, or is not every instance replaced with `userrole`? Could it be deleted?

Reply:
Ohhh good point! our workflow tests still use this method but I think we can work on removing it as part of our oso migration.